pkg/cmd/kind/delete/cluster/deletecluster.go

```diff
@@ -74,5 +74,6 @@ func deleteCluster(logger log.Logger, flags *flagpole) error {
 	if err := provider.Delete(flags.Name, flags.Kubeconfig); err != nil {
 		return errors.Wrapf(err, "failed to delete cluster %q", flags.Name)
 	}
+	logger.V(0).Infof("Deleted cluster: %q", flags.Name)
```

Comment:
Sorry, but this is not what we discussed adding in #2924.

We should log when we take specific actions like deleting node containers or if the kubeconfig is modified.
This long line doesn't add any additional information versus the log line above on line 73.

Reply:
I've amended my PR. I don't think I can log the deletion of specific nodes unless I modify the `Provider.DeleteNodes()` which doesn't seem to have access to the logging interface.

Reference(s):
- https://github.com/kubernetes-sigs/kind/blob/main/pkg/cluster/internal/providers/provider.go#L41